lib/engine/game/g_18_mag/game.rb

```diff
@@ -683,30 +683,38 @@ def revenue_for(route, stops)
                        0
                      end
 
+          ciwl_add = ciwl_delta if new_major? && red_to_red_route?(route)
```

Comment:
re: #10876

@philcampeau this probably needs to be

```
ciwl_add =  new_major? && red_to_red_route?(route) ? ciwl_delta : 0
```

Reply:
100% correct. I've submitted a fix